app/javascript/packages/document-capture/context/acuant.tsx

```diff
@@ -8,7 +8,9 @@ import SelfieCaptureContext from './selfie-capture';
 /**
  * Global declarations
  */
-declare let AcuantJavascriptWebSdk: AcuantJavascriptWebSdkInterface; // As of 11.7.0, this is now a global object that is not on the window object.
+// As of 11.7.0, Acuant provides global objects that are not on the window object. We (identity-idp)
```

Comment:
I'm a bit confused by this, since I equate "window" and "global" in the context of a browser. How are they different? Or is this something in a worker?

>In a web browser, any code [...] has a [Window](https://developer.mozilla.org/en-US/docs/Web/API/Window) as its global object.

https://developer.mozilla.org/en-US/docs/Glossary/Global_object

Reply:
This is a good point. I do know that Acuant uses workers, but I didn't think too hard about. I was leaning on the writeup [here](https://github.com/18F/identity-idp/pull/6649) pretty heavily. Specifically:

> In 11.7.0, this is no longer the case -- these objects are injected into the browser's global scope, but not added to the window object. This introduced three breaking difficulties: where to "find" the correct resources; the assumptions of our frontend test suite; and the assumptions of our type annotations.

Reply:
Is it at all possibly linked to the shadow DOM? Maybe the shadow DOM's global is not the window?

It sounds like from the documentation that `window === global`. But the history and comments in our code says otherwise. I don't feel I understand this level of how it all works (though I'd like to learn).

Reply:
It's quite likely we were just mistaken in some of that older code?

What happens if this line of code is removed altogether?

Reply:
I removed it and a couple other lines complained about it missing. I updated them the best I could guess and it...seems fine? I do think there are still some problematic comments / code like [here](https://github.com/18F/identity-idp/blob/b03f9aa7f1b53086dd0e8bfa394af88f0832874d/app/javascript/packages/document-capture/context/acuant.tsx#L191-L199) and [here](https://github.com/18F/identity-idp/blob/b03f9aa7f1b53086dd0e8bfa394af88f0832874d/app/javascript/packages/document-capture/context/acuant.tsx#L162-L180) and probably others. Made a ticket for that: [LG-12726](https://cm-jira.usa.gov/browse/LG-12726).

But I think fixing them would be a bigger lift than this work was originally meant to be, and should probably be ticketed and prioritized, as changing it offhand could break the SDK.